.gitignore

```diff
@@ -1,10 +1,13 @@
 build/
 dist/
 doc/_build
+pystan/tests/cached_models
```

Comment:
Is this for your local user environment? Does everyone need this?

Reply:
Yeah, I mean if you do

    pip install -e .

And then run the tests, you get cached models folder which is only a local folder. (We reuse some models)